mLRS/Common/hal/tx-hal-FRM303-f070cb.h

```diff
@@ -264,7 +264,7 @@ void systembootloader_init(void)
 #define FIVEWAY_ADC_IO            IO_PA7 // ADC1_IN7
 #define FIVEWAY_ADC_CHANNELx      LL_ADC_CHANNEL_7
 
-#ifdef DEVICE_HAS_I2C_DISPLAY_ROT180
+#if defined DEVICE_HAS_I2C_DISPLAY_ROT180 || defined DEVICE_HAS_I2C_DISPLAY
```

Comment:
nitpicky, could you change this to `#if defined DEVICE_HAS_I2C_DISPLAY || defined DEVICE_HAS_I2C_DISPLAY_ROT180` like in the above, pl?

Reply:
Sure, done.